.magento.app.yaml

```diff
@@ -55,6 +55,9 @@ web:
             allow: true
             scripts: false
             passthru: "/front-static.php"
+            rules:
+                /static/(version\d*/)?(?<resource>.*)$:
+                    passthru: "/static.php?resource=$resource"
```

Comment:
This should probably be:

```yaml
^/static/version\d+/(?<resource>.*)$:
  passthru: "/static/$resource"
```

The original from `nginx.conf.sample` is:

```
location ~ ^/static/version {
    rewrite ^/static/(version\d*/)?(.*)$ /static/$2 last;
}
```

But that regular expression doesn't make sense: (1) `\d*` should be `\d+` as you actually want a number in there and (2) the version group should not be optional.

That other part of the configuration:

```
if (!-f $request_filename) {
    rewrite ^/static/(version\d*/)?(.*)$ /static.php?resource=$2 last;
}
```

Is to support lazy static asset generation in development mode, we have no use for that (and given that the `passthru` above has been pointing to a non-existent file all this time, it sounds like nobody currently cares).